thread.c

```diff
@@ -2678,13 +2698,17 @@ thread_io_close_notify_all(struct rb_io *io)
 size_t
 rb_thread_io_close_interrupt(struct rb_io *io)
 {
+    if (io->fork_generation != GET_VM()->fork_gen) {
+
```

Comment:
Is something missing here?

Reply:
Ah thanks for calling that out, actually it's now handled later in `rb_io_blocking_operations`.